api/v1alpha1/jwt_types.go

```diff
@@ -81,3 +94,9 @@ type ClaimToHeader struct {
 	// to separate the JSON name path.
 	Claim string `json:"claim"`
 }
+
+// JWTFromHeader defines an HTTP header location to extract JWT token
+type JWTFromHeader struct {
+	Name        string `json:"name"`
+	ValuePrefix string `json:"valuePrefix"`
```

Comment:
nit:
 * ValuePrefix is optional, should be pointer
 * Can we add some comments here to explain what ValuePrefix is?

Reply:
sure @zhaohuabing